svelte.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
     adapter: adapter(),
 
     // hydrate the <div id="svelte"> element in src/app.html
-    target: '#svelte',
+    // target: '#svelte',
```

Comment:
[question](https://conventionalcomments.org/#labels): why did you comment it out?

Reply:
I've received `> config.kit.target is no longer required, and should be removed`.

Reply:
I see, an update to SvelteKit